docs/resources/shadow.md.erb

```diff
@@ -5,7 +5,7 @@ platform: linux
 
 # shadow
 
-Use the `shadow` InSpec audit resource to test the contents of `/etc/shadow`, which contains password details that are readable only by the `root` user. `shadow` is a [plural resource](https://www.inspec.io/docs/reference/glossary/#plural_resource). It functions by performing searches across multiple entries in the shadow file.
+Use the `shadow` InSpec audit resource to test the contents of `/etc/shadow`, which contains password details that are readable only by the `root` user. `shadow` is a [plural resource](https://www.inspec.io/docs/reference/glossary/#plural_resource). Like all plural resources, it functions by performing searches across multiple entries in the shadow file.
```

Comment:
This sentence means that "All plural resources search across multiple files _in the shadow directory_."

Reply:
It's not a directory, it is a file.

Reply:
Do all plural resources perform searches across multiple entries _in the shadow file_?